controller/namedusers_blackbox_test.go

```diff
@@ -154,49 +171,55 @@ func (s *NamedUsersControllerTestSuite) TestBan() {
 			t.Run("other service", func(t *testing.T) {
 				// Another service account can't deprovision
 				svc, ctrl := s.SecuredServiceAccountController(testsupport.TestTenantIdentity)
-				test.BanNamedusersForbidden(s.T(), svc.Context, svc, ctrl, userToBan.Identity().Username)
+				test.BanNamedusersForbidden(t, svc.Context, svc, ctrl, userToBan.Identity().Username)
 
 			})
 
 			t.Run("missing token", func(t *testing.T) {
 				// If no token present in the context then fails too
 				_, ctrl := s.SecuredServiceAccountController(testsupport.TestOnlineRegistrationAppIdentity)
-				test.BanNamedusersForbidden(s.T(), nil, nil, ctrl, userToBan.Identity().Username)
+				test.BanNamedusersForbidden(t, nil, nil, ctrl, userToBan.Identity().Username)
 			})
 
 			t.Run("regular user", func(t *testing.T) {
 				// Regular user can't deprovision either
 				svc, ctrl := s.SecuredController(*s.Graph.CreateUser().Identity())
-				test.BanNamedusersForbidden(s.T(), svc.Context, svc, ctrl, userToBan.Identity().Username)
+				test.BanNamedusersForbidden(t, svc.Context, svc, ctrl, userToBan.Identity().Username)
```

Comment:
you don't need to make this change anymore, since a recent release of `testify`, the `s.T()` func works in subtests as well.

Reply:
Ah, ok. Didn't know that.